test/solidity/Facets/DeBridgeFacet.t.sol

```diff
@@ -76,167 +71,61 @@ contract DeBridgeFacetTest is DSTest, DiamondTest {
             ? IDeBridgeGate(DEBRIDGE_GATE).globalFixedNativeFee()
             : chainConfig.fixedNativeFee;
-        executionFee = 10**usdc.decimals();
+        uint256 executionFee = 1 * 10**usdc.decimals();
 
-        validDeBridgeData = DeBridgeFacet.DeBridgeData(
+        deBridgeData = DeBridgeFacet.DeBridgeData(
             "",
-            false,
             nativeFee,
+            false,
             0,
             DeBridgeFacet.SubmissionAutoParamsTo(
                 executionFee,
                 REVERT_IF_EXTERNAL_FAIL,
-                abi.encodePacked(DAI_HOLDER),
+                abi.encodePacked(USER_RECEIVER),
                 ""
             )
         );
     }
 
-    function testRevertToBridgeTokensWhenSendingAmountIsZero() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.minAmount = 0;
-
-        vm.expectRevert(InvalidAmount.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenReceiverIsZeroAddress() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.receiver = address(0);
-
-        vm.expectRevert(InvalidReceiver.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenSenderHasNoEnoughAmount() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        usdc.transfer(DAI_HOLDER, usdc.balanceOf(USDC_HOLDER));
-
-        vm.expectRevert(abi.encodeWithSelector(InsufficientBalance.selector, 10 * 10**usdc.decimals(), 0));
-        deBridgeFacet.startBridgeTokensViaDeBridge(validBridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenSendingNoEnoughNativeAsset() public {
-        vm.startPrank(USDC_HOLDER);
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.sendingAssetId = address(0);
-        bridgeData.minAmount = 3e18;
-
-        vm.expectRevert(InvalidAmount.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge{ value: 2e18 }(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
+    function testBase_Revert_BridgeWithInvalidDestinationCallFlag() public override {
+        // facet supports destination call
     }
 
-    function testRevertToBridgeTokensWhenInformationMismatch() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
+    function testBase_CanBridgeTokens_fuzzed(uint256 amount) public override {
```

Comment:
You could simplify this override similar to the one in stargate facet:

    function testBase_CanBridgeTokens_fuzzed(uint256 amount) public override {
        // fails otherwise with "slippage too high" from Stargate router contract
        vm.assume(amount > 100);
        super.testBase_CanBridgeTokens_fuzzed(amount);
    }

Reply:
updated the test